client/js/autocompletion.js

```diff
@@ -177,16 +177,34 @@ function fuzzyGrep(term, array) {
 	return results.map((el) => [el.string, el.original]);
 }
 
-function completeNicks(word, isFuzzy) {
+function rawNicks() {
 	const users = chat.find(".active .users");
+
+	// If there are no users, this is probably a private chat
+	// -- use the title.
+	if (users.length) {
+		return users.data("nicks");
+	}
+
+	const otherUser = chat.find(".chan.active");
+	// Lobbies and private chats do not have an user list
```

Comment:
This should check if active channel is a query then, because we wouldn't want to autocomplete lobbies and custom windows.

Reply:
I mean... I can try to figure that out, but is there any harm in it?